public/app/ui/pages/search/SearchPage.less

```diff
@@ -1,131 +1,29 @@
 article.search {
-    .search-tips {
-        h2 {
-            font-size: @font-size-h4;
-            margin-top: 0;
-        }
-    }
+    .search-panel {
+        transition: width ease .5s;
 
-    .result {
```

Comment:
Most of this file has been split out into other files, e.g. `SearchResultsComponent.less`.